cookbooks/convection-box/tutorial-onset-of-convection/README.md

```diff
@@ -0,0 +1,86 @@
+Author: Lorraine J. Hwang, Ian Rose, Juliane Dannberg and the ASPECT development community
+
+# Introduction to ASPECT
+
+This notebook is based on tutorials by J. Dannberg that provide a basic introduction to ASPECT.
```

Comment:
Do we have a better place to put notebooks/tutorials? It seems weird to me to hide them inside the convection-box folder.

Reply:
This was part of a broader discussion to keep notebooks as close as possible to the related prm's such that they stay current.